server/chat-commands/core.ts

```diff
@@ -16,6 +16,7 @@
 
 /* eslint no-else-return: "error" */
 import {Utils} from '../../lib/utils';
+import {User, UserSettings} from '../users';
```

Comment:
```suggestion
type UserSettings = import('../users').UserSettings;
```

At some point we should investigate making sure sucrase is recent enough for `import type`. Until then, this, I guess.